hubblestack/extmods/modules/nebula_osquery.py

```diff
@@ -1390,32 +1391,100 @@ def _parse_log(path_to_logfile,
                             log.info("Found few residue logs, updating the data object")
                             event_data += residue_events
                         file_offset = 0  # Reset file offset to start of file in case original file is rotated
-                _set_cache_offset(path_to_logfile, file_offset)
+                _set_file_offset(path_to_logfile, file_offset)
             else:
                 log.error('Unable to open log file for reading: {0}'.format(path_to_logfile))
     else:
         log.error("Log file doesn't exists: {0}".format(path_to_logfile))
 
     return event_data
 
-
-def _set_cache_offset(path_to_logfile, offset):
+def _set_file_offset(path_to_logfile, offset):
```

Comment:
Ensure that the previous function wasn't called from other places.

Reply:
Checked